base/keycloak-deployment/db.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: keycloak-db
+spec:
+  serviceName: keycloak-db
+  selector:
+    matchLabels:
+      app: keycloak-db
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: keycloak-db
+    spec:
+      containers:
+        - name: keycloak-db
+          image: quay.io/centos7/postgresql-10-centos7
```

Comment:
Why are we using postgres 10, cannot we use newer version?

Reply:
I used Postgress 10 as seen in `base/rhbk/rhbk-db.yaml`.
I changed the image for newer.